app/src/main/java/com/example/next_show/fragments/FeedFragment.java

```diff
@@ -271,8 +265,12 @@ public void onSuccess(JsonHttpResponseHandler.JSON json) {
                 // finds show in adapter
                 int index = adapter.search(jsonObj.getString("id"));
 
-                // updates the show images
-                adapter.updateShowImage(index, path);
+                // update show in FeedFragment -> same show list as adapter!
+                Show show = showsList.get(index);
```

Comment:
I would just search the list of shows in here and not search in the adapter.  Then, when you get the index for the list of shows in here, you tell the adapter to update that